kairon/shared/data/processor.py

```diff
@@ -5469,6 +5470,23 @@ def get_bot_settings(bot: Text, user: Text):
             settings = BotSettings(bot=bot, user=user).save()
         return settings
 
+    @staticmethod
+    def add_demo_request(**kwargs):
+        try:
+            logs = DemoRequestLogs(
+                first_name=kwargs.get("first_name"),
+                last_name=kwargs.get("last_name"),
+                email=kwargs.get("email"),
+                phone=kwargs.get("phone", None),
+                message=kwargs.get("message", None),
+                status=kwargs.get("status", DEMO_REQUEST_STATUS.REQUEST_RECEIVED.value),
+                recaptcha_response=kwargs.get("recaptcha_response", None),
+            ).save()
+        except Exception as e:
+            logging.error(str(e))
+            raise AppException(e)
```

Comment:
remove raise error part
```suggestion
            raise AppException(e)
```